.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,45 @@
+name: Update Docs
+on:
+  pull_request:
+
+jobs:
+  update_dfx_json_schema:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: ["1.60.0"]
+        os: [ ubuntu-20.04 ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Rust
```

Comment:
more of a fyi, cause I'm not 100% certain here (help @adamspofford-dfinity), but I think this step is unnecessary since we have `rust-toolchain.toml` file - cargo should be able to use the correct version here

to verify, you might want to try `rustc --version` instead and see if its `1.60.0`